### DISCUSSION TITLE
What to look for when buying a laser?

### DISCUSSION THREAD

**Discussion by Setcover** [bhowiebkr/laser-level-webcam#25] [Q&A]
How can you tell whether a laser has good thermal properties before buying?

Attached are measurements from the laser and camera stationary on a granite slab on the concrete floor slab of a building. Starting at around measurement 125 the laser receives a little air flow. My laser (plastic housing) is obviously not suitable for this use, but how do I recognize a laser that might be suitable?

Brightness `histo.max()` fluctuations in the range of 117 to 131 can be observed too.
https://github.com/bhowiebkr/laser-level-webcam/blob/f990425975e568733e7ef49ef476610c8686a055/src/Workers.py#L155

![Unbenannt](https://github.com/bhowiebkr/laser-level-webcam/assets/20210370/f89ec7f8-fdd7-4a5e-af8f-35fbfb1e2ec0)

**Comment by miked243** [comment]
hello, this might be a little late for you but off the shelf I would just look for something with a metal housing and possibly even a heat sink. but assembling your own laser is quite easy and allows you to select the best parts for your set up.
